deployment/terraform/firewall.tf

```diff
@@ -0,0 +1,67 @@
+# allow all inbound traffic to the load balancer on port 443
+resource "aws_security_group_rule" "oar_alb_ingress_https" {
+  type             = "ingress"
+  from_port        = 443
+  to_port          = 443
+  protocol         = "tcp"
+  cidr_blocks      = ["0.0.0.0/0"]
+  ipv6_cidr_blocks = ["::/0"]
+
+  security_group_id = "${aws_security_group.alb.id}"
+}
+
+# allow outbound traffic from the alb to the ecs cluster on the app port
```

Comment:
These comments are helpful, but I think it is more accurate to associate the security group rules with the ECS service vs. the ECS cluster (if there were two services, we could have two different security groups with each).